java/pgjdbc_using_dsql_connector/setup-env.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
I would prefer if we could remove this - I think specifying the necessary and optional environment variables in the `README.md` file as you've already done is sufficient.

We want to try to keep the number of files in these samples as small as possible, and this file has a fair amount of complexity which would need to be tested by hand if we (or community members) add a new variable or make other changes in the future.

It also seems easier to stick with cross-platform tooling that we know will work on Windows, given Java has great support there and customers may be trying to run these samples in an environment that doesn't support Bash.